pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -226,14 +226,19 @@ def replay_genargs_handle_dashI(arg: str, target_install_dir: str) -> str | None
         The new argument, or None to delete the argument.
     """
     assert arg.startswith("-I")
-    if (
-        str(Path(arg[2:]).resolve()).startswith(sys.prefix + "/include/python")
-        and "site-packages" not in arg
-    ):
-        return arg.replace("-I" + sys.prefix, "-I" + target_install_dir)
+
     # Don't include any system directories
     if arg[2:].startswith("/usr"):
```

Comment:
Just curious, could this system directories case can be removed since it is already tackled by the `sys.base_prefix`?

Maybe there are other reasons to remove system directories than the Python include though.

Reply:
I am not sure to be honest, we will have to check. Some codes in `pywasmcross.py` are very old so no one remembers the context 😅